### ISSUE TITLE
Ubuntu 22.04 "ppa does not have release file" error

### ISSUE THREAD

**Issue by thereticle** [OpenShot/openshot-qt#4894]
**Describe the bug:**
After adding ppa repository and updating apt, I have:
Errore:9 https://ppa.launchpadcontent.net/openshot.developers/ppa/ubuntu jammy Release  404  Not Found [IP: 185.125.190.52 443]
E: Il repository "https://ppa.launchpadcontent.net/openshot.developers/ppa/ubuntu jammy Release" non ha un file Release.
N: L'aggiornamento da tale repository non può essere eseguito in modo sicuro ed è quindi disabilitato come impostazione predefinita.

that in English is: 
Error:9 https://ppa.launchpadcontent.net/openshot.developers/ppa/ubuntu jammy Release  404  Not Found [IP: 185.125.190.52 443]
E: Repository "https://ppa.launchpadcontent.net/openshot.developers/ppa/ubuntu jammy Release" does not have a Release file.
N: Updating from such a repository cannot be performed safely and is therefore disabled by default.

This happens with both ppas: stable and daily

**Steps to reproduce the behavior:**
1. Add the ppa
2. Update apt 

**System Details:**
- Operating System / Distro: Xbuntu 22.04

**Screenshots:** (Optional)

![immagine](https://user-images.githubusercontent.com/20416601/185879347-9afac1b2-37a8-4b89-92b8-a59af08018ac.png)

**Comment by rl-musik** [comment]
Same problem: 

 ​ ​ ​ ​ OpenShot and UbuntuStudio 22:
 ​ ​ ​ ​ [https://github.com/OpenShot/openshot-qt/issues/4847#issuecomment-1200118291]
 
 ​ ​ ​ ​ PPA-Schlüssel im veralteten Schlüsselbund trusted.gpg gespeichert:
 ​ ​ ​ ​ [https://github.com/OpenShot/openshot-qt/issues/4905#issue-1361086008]

 ​ ​ ​ ​ Appears incompatible with Ubuntu 22.04:
 ​ ​ ​ ​ [https://github.com/OpenShot/openshot-qt/issues/4878#issuecomment-1236295964]

 ​ ​ ​ ​    ▶      ​ ​ ​ ​ Note: meanwhile some updates appeared 
 ​ ​ ​  ​ ​ ​     ​ ​ ​     ​ ​ ​   (automatically installed by the Update Manager / Software Updater 
  ​ ​ ​ ​ ​ ​     ​ ​ ​     ​ ​ ​   (resp. `mintupdate `(Aktualisierungsverwaltung)), 
 ​ ​ ​  ​ ​ ​     ​ ​ ​     ​ ​ ​   or with `sudo apt update` and `sudo apt upgrade`)
   ​ ​ ​     ​ ​ ​     ​ ​ ​

**Comment by thereticle** [comment]
> 

Check that ppa's address does not contain any trailing spaces.

**Comment by rl-musik** [comment]
> > 
> 
> Check that ppa's address does not contain any trailing spaces.

What please are 'trailing spaces' ?

**Comment by thereticle** [comment]
I am not sure if this is the cause of the problem, but I realised that when
I copied the ppa string, I also copied a space at the end of the string.

I manually inserted the ppa into the repositories by copying the string:
"deb
https://ppa.launchpadcontent.net/openshot.developers/libopenshot-daily/ubuntu
jammy main"

from the page:
https://launchpad.net/~openshot.developers/+archive/ubuntu/libopenshot-daily

Trying to solve the issue, I saw that the copied string contained a space
at the end:
"deb
https://ppa.launchpadcontent.net/openshot.developers/libopenshot-daily/ubuntu
jammy main "

By removing it, the error disappeared, and now it works as expected.

But, as I said before, I am not sure this was the cause.

Sorry for my bad English. :)

Il mar 6 set 2022, 07:23 Putzwürfel ***@***.***> ha scritto:
